pkg/ignore/ignore_test.go

```diff
@@ -27,6 +28,33 @@ func TestIgnore_Match(t *testing.T) {
 	}
 }
 
+// Test all default ignore files, except for .git/info/exclude, since
+// that uses a .git directory that we cannot check in.
+func TestIgnoreDefaultIgoreFiles_Match(t *testing.T) {
+	// Change into testdata directory for this test
+	// since paths are relative
+	err := os.Chdir("testdata")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
```suggestion
	assert.NoError(t, err)
```
I'd recommend using the `assert` package instead of panicking

Reply:
Thanks Caitlin! I apologize, this is the first Go project I've worked on. I've updated the code, please re-check if possible.

Reply:
No worries! The `assert` package isn't a part of the standard library, but it does make writing tests a lot easier. Looks good, thanks for the contribution!